client/components/ReviewRating/WriteReview/WriteReview.jsx

```diff
@@ -1,10 +1,10 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 
```

Comment:
This line break isn't necessary. Group your imports together by type
1. External (i.e. 'react', 'prop-types', 'axios')
2. Internal (i.e. components, utility functions, etc.)
3. CSS imports

Reply:
Fixed.